misc/run-phoenix-release-cpu.sh

```diff
@@ -12,5 +12,4 @@ cd "$SLURM_SUBMIT_DIR"
 echo "Running in $(pwd):"
 
 . ./mfc.sh load -c p -m gpu
-./mfc.sh test -j $(nproc) -b mpirun -a
-
+./mfc.sh test -j $(nproc) -a -- -b mpirun
```

Comment:
can you comment on what `--` does?

Reply:
It forwards arguments to `./mfc.sh run` (called by `./mfc.sh test`). This means we no longer have to redocument all of these options as options of `./mfc.sh test`, too. We also no longer have to manually forward each option.

Reply:
I just changed this line to:
```console
./mfc.sh test -j $(nproc) -a -- -c phoenix
```